srcalc/src/test/java/gov/va/med/srcalc/service/MockAdminService.java

```diff
@@ -0,0 +1,63 @@
+package gov.va.med.srcalc.service;
+
+import gov.va.med.srcalc.domain.model.*;
+
+import java.util.List;
+import java.util.Set;
+
+/*
+ * Created to support the EditRiskModel which needs to call getAllSpecialties()
+ * Implement other methods as needed.
+ * 
+ */
+public class MockAdminService extends MockModelService implements AdminService  {
+
+    public MockAdminService() 
+    {        
+    }
+    
+    @Override
+    public List<AbstractVariable> getAllVariables() {
+        return null;
+    }
+
+    @Override
+    public AbstractVariable getVariable(String key)
+            throws InvalidIdentifierException {
+        return null;
+    }
+
+    @Override
+    public void saveVariable(AbstractVariable variable) {
+        // not implemented        
+    }
+
+    @Override
+    public void saveRiskModel(RiskModel model) {
```

Comment:
I don't see the point of having all of these methods in here if they aren't implemented or used anywhere in the code.